pzsd_bot/cogs/points.py

```diff
@@ -28,6 +30,8 @@
 
 
 class Points(Cog):
+    leaderboard = SlashCommandGroup("leaderboard", "Display point leaderboards.")
```

Comment:
If constant should be capitalized

Reply:
As discussed in discord, it's a decorator, and thus would be a little weird to capitalize.